find_me_a_sandwich/app/controllers/favorites_controller.rb

```diff
@@ -0,0 +1,22 @@
+class FavoritesController < ApplicationController
+  def show
+    params[:id] ||= current_user.id if user_signed_in?
+
+    @favorites = Favorite.where(user_id: params[:id])
```

Comment:
In your view, you load favorites' menu items as well.  At present, for every menu item you load, you query the database.  You can fix this by including the association.

```ruby
@favorites = Favorite.includes(:menu_item).where(user_id: params[:id])
```